gcc/config/riscv/riscv.c

```diff
@@ -2420,7 +2427,7 @@ riscv_fpr_return_fields (const_tree valtype, tree fields[2])
       if (!SCALAR_FLOAT_TYPE_P (TREE_TYPE (field)))
 	return 0;
 
-      if (i == ARRAY_SIZE (fields))
+      if (i == 2)
```

Comment:
Can you undo this change?  They are equivalent, but the previous one makes the intent clearer.  (I think this is a merge issue.)